org.eclipse.buildship.ui/src/main/java/org/eclipse/buildship/ui/wizard/project/GradleProjectWizardPage.java

```diff
@@ -99,6 +99,19 @@ private void bindToConfiguration() {
             public void modifyText(ModifyEvent e) {
                 File projectDir = FileUtils.getAbsoluteFile(GradleProjectWizardPage.this.projectDirText.getText()).orNull();
                 getConfiguration().setProjectDir(projectDir);
+                // TODO currently a
```

Comment:
Implement the validation properly. 
1. Add the validator into the org.eclipse.buildship.core.util.binding.Validators class.
2. Implement an `and()` validator in the same class. 
3. Combine the existing and the new validator in the controller class with the `and()` validator